kolibri/core/assets/src/api_resource.js

```diff
@@ -244,30 +421,23 @@ class Resource {
   }
 
   /**
-   * Get a model by id - this will either return a Model that has already been instantiated,
-   * or will instantiate a model
-   * that can later be fetched.
+   * Get a model by id
    * @param {String} id - The primary key of the Model instance.
-   * @param {Object} [data={}] - Optional additional data to pass into the Model.
    * @returns {Model} - Returns a Model instance.
    */
-  getModel(id, data = {}) {
-    let model;
+  getModel(id) {
     if (!this.models[id]) {
-      Object.assign(data, { [this.idKey]: id });
-      model = new Model(data, this);
-    } else {
-      model = this.models[id];
+      return null;
```

Comment:
Please revert this change - `createModel` is just creating a model on the client side, this breaks the API for `getModel` as used in other parts of the application. The point of `getModel` is to retrieve an object that either is the fully synced object that corresponds to the passed id, or an unsynced version of that object that later can have fetch called on it.